### PR TITLE
Feat/msdk 23 sonar cloud

### PR DESCRIPTION
**Pull request by shuffledex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +20,40 @@ Things included in the repo:
...
-- `yarn build:docs` builds a documentation page from tsdoc comments in the code. Outputs to `docs/` directory
+- `yarn generate:polkadot-types` generates polkadot types to interact with the latest Polymesh blockchain version
 - `yarn test` runs tests and outputs the coverage report
+- `yarn build:ts` compiles typescript files into javascript and type declarations. Outputs to `dist/` directory
+- `yarn build:docs` builds a documentation page from tsdoc comments in the code. Outputs to `docs/` directory
 - `yarn commit` runs the commit formatting tool (should replace normal commits)
 - `yarn semantic-release` runs semantic release to calculate version numbers based on the nature of changes since the last version (used in CI pipelines)
 - `yarn lint` runs the linter on all .ts files and outputs all errors
+- `yarn format` runs prettier-eslint on all .ts files and rewrites the files with well formatted code
+- `yarn ts-node` compiles typescript files to be run seamlessly in a Node environment
```
Comment (monitz87) [inline README.md:31]: run a ts script without prior compilation

File: README.md [hunk 2]
```diff
@@ -19,9 +20,40 @@ Things included in the repo:
...
+
+## Usage
+
+Connect to a Polymesh node with an account via an suri
```
Comment (monitz87) [inline README.md:35]:
```suggestion
Connect to a Polymesh node with an account URI
```

File: sonar-project.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
+sonar.organization=polymathnetwork-1
+sonar.projectKey=PolymathNetwork_polymesh-sdk
+sonar.sources=.
+sonar.javascript.environments=node
```
Comment (monitz87) [inline sonar-project.properties:4]: Shouldn't all these be `sonar.typescript.blah`?

Comment (shuffledex) [reply sonar-project.properties:4]: It dependes.
For coverage:  `sonar.javascript.lcov.reportPaths | Comma-delimited list of paths to LCOV coverage report files. Paths may be absolute or relative to project root.`
For TSLint: `sonar.typescript.tslint.reportPaths`

**Pull request merged**